sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-response/_tests_/ece-response-helper.ts

```diff
@@ -72,11 +94,11 @@ async function findOrCreateInstitutionLocation(
   if (!institutionLocation) {
     const newInstitutionLocation = createFakeInstitutionLocation(institution);
     newInstitutionLocation.institutionCode = institutionCode;
-    institutionLocation = await e2eDataSources.institutionLocation.save(
-      newInstitutionLocation,
-    );
+    institutionLocation = newInstitutionLocation;
   }
-  return institutionLocation;
+  institutionLocation.integrationContacts = options?.initialValues
```

Comment:
@dheepak-aot I remember. In our discussion, you were saying about a scenario where `integrationContacts` is null. is that not handled here? Or planning for the future?

Reply:
We can plan if that is required in future @ann-aot

Reply:
but null doesn't mean an empty array right? Anyway resolving the conversation as it is not in the scope of the PR